tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I had to turn on the --resolveJsonModule switch for importing json files.
I first looked for commandline options in "tsd --help" but there are none.